spec/ripple/conflict/resolver_spec.rb

```diff
@@ -8,10 +8,10 @@ module Conflict
       end
 
       describe 'calling the lambda returned by .to_proc' do
-        let(:sibling_1) { stub(:data => { '_type' => 'User' } ) }
-        let(:sibling_2) { stub(:data => { '_type' => 'User' } ) }
+        let(:sibling_1) { stub(:data => { '_type' => 'User' }, :content_type => 'application/json') }
```

Comment:
@dtykocki I guess if we're ensuring backwards compatibility, you'd like to see that all existing tests can pass. Modifying the existing tests for such a fundamental library like Ripple makes me a little nervous.